spec/disyuntor_spec.rb

```diff
@@ -0,0 +1,308 @@
+require "minitest/autorun"
+require_relative "../lib/disyuntor"
+
+describe Disyuntor do
+  let(:circuit) { Disyuntor.new }
+
+  describe :new do
+    it "be closed by default" do
```

Comment:
Missing a "should" here? Otherwise maybe it should read `it "is closed by default"`